### ISSUE TITLE
version 3.2.0 crashes

### ISSUE THREAD

**Issue by volkerrichert** [tusharmath/node-config-ts#81]
**Describe the bug:**
postinstall crashes with 

```
$ node-config-ts
✨  Done in 2.51s.
➜  bap-esim git:(develop) ✗ yarn install
yarn install v1.22.19
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > @type-cacheable/redis-adapter@13.0.0" has unmet peer dependency "@redis/client@^1.1.0".
warning " > filter-async-rxjs-pipe@1.0.1" has unmet peer dependency "es6-promise@>= 4.2.6 < 5".
warning " > node-config-ts@3.2.0" has unmet peer dependency "@types/webpack@4.x".
warning " > node-config-ts@3.2.0" has unmet peer dependency "webpack@4.x".
warning "wsdl-tsclient > eslint-plugin-prettier@4.2.1" has unmet peer dependency "prettier@>=2.0.0".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
$ node-config-ts
node:internal/modules/cjs/loader:988
  throw err;
  ^

Error: Cannot find module '../src/createTypedefs'
Require stack:
- /Users/volker/PhpstormProjects/bap-esim/node_modules/node-config-ts/bin/cli
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:985:15)
    at Function.Module._load (node:internal/modules/cjs/loader:833:27)
    at Module.require (node:internal/modules/cjs/loader:1057:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/Users/volker/PhpstormProjects/bap-esim/node_modules/node-config-ts/bin/cli:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1155:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1209:10)
    at Module.load (node:internal/modules/cjs/loader:1033:32)
    at Function.Module._load (node:internal/modules/cjs/loader:868:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/volker/PhpstormProjects/bap-esim/node_modules/node-config-ts/bin/cli'
  ]
}

```

**Steps To Reproduce:**
install version 3.2.0

**node-config-ts version:** 3.2.0. 

3.1.0 works fine

**Comment by DonIsaac** [comment]
@tusharmath the publish pipeline seems broken, could this be related?

**Comment by na-ji** [comment]
Same problem on v3.3.0.

**Comment by tusharmath** [comment]
This is fixed in 3.2.1
